cl/scrapers/utils.py

```diff
@@ -555,3 +556,42 @@ def save_response(site: AbstractSite) -> None:
 
     content_name = f"{base_name}.{extension}"
     storage.save(content_name, ContentFile(content))
+
+
+def check_duplicate_ingestion(local_path_name: str) -> None:
+    """
+    Send an error log to Sentry if a filename has a high repetition count
+
+    S3 filenames / filepaths for opinions and oral arguments are created using
+    created via `cl.lib.storage.get_name_by_incrementing`. The name is composed
+    of the document's filed date or argued date, the case name and the file
+    extension. `get_name_by_incrementing` adds a counter if the name already
+    existed
+
+    For example, this file name means there are 2 documents for that case name,
+    date and extension combination
+    'pdf/2025/06/05/state_v._walsh_1.pdf'
+
+    :param local_path_name: filepath of the file in S3
+    :return None
+    """
+    # we may get the same file path for:
+    # - a cluster of opinions, which are actually different documents for the
+    #     same date and case
+    # - a common case name in the same date, for example, "State v. Doe"
```

Comment:
can we clean this up.  

Id like to make the first line one line and we should move the comment here in to the docstrings ti think.  Or atleast part of it.